datashare-app/src/main/java/org/icij/datashare/web/OpenApiResource.java

```diff
@@ -0,0 +1,22 @@
+package org.icij.datashare.web;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.inject.Singleton;
+import io.swagger.v3.core.util.Json31;
+import io.swagger.v3.oas.models.OpenAPI;
+import net.codestory.http.annotations.Get;
+import net.codestory.http.annotations.Prefix;
+import net.codestory.http.payload.Payload;
+import org.icij.swagger.FluentReader;
+
+import static org.icij.swagger.ClassUtils.findAllClassesUsingClassLoader;
+
+@Singleton
+@Prefix("/api/openapi")
+public class OpenApiResource {
+    @Get()
+    public Payload get() {
```

Comment:
Shouldn't we annotation this endpoint as well?

Reply:
haha yes. I'm adding it.